src/plugin/index-more/src/java/org/apache/nutch/indexer/more/MoreIndexingFilter.java

```diff
@@ -82,6 +87,20 @@
   private HashMap<String, String> mimeMap = null;
   private boolean mapMimes = false;
   private String mapFieldName;
+  
+  /** Date-styles used to parse date. */
+  private String[] dateStyles = new String[] {
```

Comment:
Not sure if we want to keep these here? It gets overwritten with the content of the config file if the additional config file loading is successful. _If_ we want to keep them I would rename them to something like `defaultDateStyles` and potentially only use the config file for additional formats.